web/docs/user-guides/reshape-data/README.md

```diff
@@ -100,8 +100,9 @@ tenzir 'export | where #schema == "suricata.alert"'
 
 Or by using type and field extractors:
 
-```bash
-tenzir 'export | where 10.10.5.0/25 && (orig_bytes > 1 Mi || duration > 30 min)'
+```
+export
+| where 10.10.5.0/25 && (orig_bytes > 1 Mi || duration > 30 min)
 ```
 
```

Comment:
Not in your diff, but: "The above example extracts connections that either have sent more than 1 MiB or lasted longer than 30 minutes" does not mention the subnet constraint.